microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -88,11 +80,11 @@ Your download is a `.zip` archive that contains an `.msixbundle` file and a `.cl
 
```

Comment:
line ~76: to avoid the "?." dup punc, change from:
```
see [What is a classic package?](https://github.com/pwa-builder/pwabuilder-windows-chromium-docs/blob/master/classic-package.md).
```
to:
```
see [What is a classic package?](https://github.com/pwa-builder/pwabuilder-windows-chromium-docs/blob/master/classic-package.md)
```
rendered: end of section:
https://github.com/zateutsch/edge-developer/blob/2482-updated-pwa-partner-center-guidance/microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md#package-your-pwa-for-the-store

Reply:
done